modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -49,6 +49,10 @@ export enum TransactionType {
 export enum AddressFormat {
   hex = 'hex',
   base58 = 'base58',
+  // format for westend addresses
```

Comment:
If this `enum` is supposed to be coin agnostic, what do you think about creating a DOT specific `AddressFormat` and [creating a union](https://stackoverflow.com/questions/52857698/extending-enum-in-typescript) of the two `enums` (since you can not extend `enums` in ts)

Reply:
Good call and ran through my mind as well and agree it should be done the way you suggested. I can do this on a follow up. I would create a ticket but you know.